webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/AnnotationPage.java

```diff
@@ -313,7 +313,7 @@ protected void onConfigure()
 
         add(new LambdaAjaxLink("showOpenDocumentModal", this::actionShowOpenDocumentDialog));
         
-        add(new LambdaAjaxLink("showPreviousDocument", t -> actionShowPreviousDocument(t))
+        add(new LambdaAjaxLink("showPreviousDocument", this::actionShowPreviousDocument)
```

Comment:
Nope. There is a reason why I used "t -> actionShowPreviousDocument(t)". It is a bug in the (de)serialization of lambda functions in the JVM. I don't find a reference right now though... but this has to stay as it was.